internal/client/github.go

```diff
@@ -105,11 +105,10 @@ func (c *githubClient) Changelog(ctx *context.Context, repo Repo, prev, current
 func (c *githubClient) getDefaultBranch(ctx *context.Context, repo Repo) (string, error) {
 	p, res, err := c.client.Repositories.Get(ctx, repo.Owner, repo.Name)
 	if err != nil {
-		log.WithFields(log.Fields{
-			"projectID":  repo.String(),
-			"statusCode": res.StatusCode,
-			"err":        err.Error(),
-		}).Warn("error checking for default branch")
+		log.WithField("projectID", repo.String()).
+			WithField("statusCode", res.StatusCode).
```

Comment:
if `err` is not `nil` is `res` also guaranteed to be not `nil`?

https://github.com/Unpackerr/xt/actions/runs/7597537660/job/20692451367#logs